python-package/lightgbm/compat.py

```diff
@@ -122,7 +122,9 @@ class _LGBMRegressorBase:  # type: ignore
 
         pass
 
+    _LGBMBaseCrossValidator = None
     _LGBMLabelEncoder = None
+    _LGBMBaseCrossValidator = None
```

Comment:
This is already included above
```suggestion
```

Reply:
ha thank you, maybe that's leftover from a bad merge conflict solve. So embarrassing 🙈 .